examples/bond/scripts/run/bond-functions.js

```diff
@@ -0,0 +1,328 @@
+const {
+  executeTransaction, convert, readGlobalStateSSC, balanceOf
+} = require('@algo-builder/algob');
+const { types } = require('@algo-builder/web');
+const { exec } = require('child_process');
+const { exit } = require('process');
+
+let newAsaInfo;
+let appInfo;
+let asaInfo;
+let buybackLsig;
+const assetID = 'asset-index';
+
+// fund account using master account
+async function fundAccount (deployer, accountAddress) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const algoTxnParams = {
+    type: types.TransactionType.TransferAlgo,
+    sign: types.SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: accountAddress,
+    amountMicroAlgos: 200e6,
+    payFlags: {}
+  };
+  await executeTransaction(deployer, algoTxnParams);
+}
+
+/**
+ * Creates DEX_i lsig, burn B_i tokens, issue B_i+1 tokens
+ * @param {Account} masterAccount
+ * @param {Account} creatorAccount
+ * @param {Account} storeManagerAccount
+ */
+async function createDex (deployer, masterAccount, creatorAccount, storeManagerAccount) {
```

Comment:
the function should take a paramter `i` - an index of the bond token to create. In the function we should assert that `i >= 1`

Reply:
done